packages/ui-components/src/components/Map/Table/useMapTable.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import { useTable, useSortBy } from 'react-table';
+import { getFormattedInfo } from '../utils';
+
+const FirstColumnCell = styled.span`
+  font-weight: 500;
+  text-align: left;
+`;
+
+const processColumns = measureOptions => {
+  const configColumns = measureOptions.map(column => {
+    return { accessor: column.key, Header: column.name };
+  });
+
+  return [
+    {
+      Header: 'Name',
+      accessor: 'name',
+      // eslint-disable-next-line react/prop-types
+      Cell: ({ value }) => <FirstColumnCell>{String(value)}</FirstColumnCell>,
+    },
+    ...configColumns,
+    { Header: 'Submission Date', accessor: 'submissionDate' },
+  ];
+};
+
+const processData = (serieses, measureData) => {
```

Comment:
In some places we're using `serieses`, and in others sticking with `measureOptions`. I'm fine with either, but could you convert them all to one of the two?